docs/notes/tutorial_index.rst

```diff
@@ -0,0 +1,52 @@
+.. _tutorial_index:
+
+Tutorial Index
+==============
+
+Kaolin provides tutorials as ipython notebooks, docs pages and simple scripts. Note that the links
+point to master.
+
+
+Detailed Tutorials
+------------------
+
+* `Deep Marching Tetrahedra <https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/tutorial/dmtet_tutorial.ipynb>`_: reconstructs a tetrahedral mesh from point clouds with `DMTet <https://nv-tlabs.github.io/DMTet/>`_, covering:
+    * generating data with Omniverse Kaolin App
+    * loading point clouds from a ``.usd`` file
+    * chamfer distance as a loss function
+    * differentiable marching tetrahedra
+    * using Timelapse API for 3D checkpoints
+    * visualizing 3D results of training
+* `Understanding Structured Point Clouds (SPCs) <https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/tutorial/understanding_spcs_tutorial.ipynb>`_: walks through SPC features, covering:
```

Comment:
So, this link leads to the jupyter notebook, but there is also the page https://kaolin--534.org.readthedocs.build/en/534/notes/spc_summary.html which is also mainly here to link to the tutorial (since it is in the tutorial section), is this redundancy on purpose, or shouldn't this link first to the spc_summary page?

Reply:
Redundancy is good. SPC and DIB-R are in the left hand side because people are likely to have questions and the primary goal is to lead them to the tutorials, which Tutorial Index is also doing. I think it is best to keep as is and redirect directly to the tutorial without and additional stop over in a redundant page.